x-pack/plugins/security_solution/public/value_list/translations.ts

```diff
@@ -131,7 +124,7 @@ export const FAILED_TO_UPLOAD_LIST_ITEM_TITLE = i18n.translate(
 );
 
 export const UPLOAD_TOOLTIP = i18n.translate('xpack.securitySolution.listItems.uploadTooltip', {
-  defaultMessage: 'All items from the file will be added as new items',
+  defaultMessage: 'All items will be added to the value list.',
```

Comment:
Hey, @nkhristinin. Leaving one more comment to share feedback from @approksiu:

```suggestion
  defaultMessage: 'All items from the file will be added to the value list.',
```